packages/blocks/src/store/test/selectors.js

```diff
@@ -134,4 +137,66 @@ describe( 'selectors', () => {
 			expect( getChildBlockNames( state, 'parent2' ) ).toEqual( [ 'child2' ] );
 		} );
 	} );
+
+	describe( 'isMatchingSearchTerm', () => {
+		const name = 'core/paragraph';
+		const blockType = {
+			title: 'Paragraph',
+			category: 'common',
+			keywords: [ 'text' ],
+		};
+
+		const state = {
+			blockTypes: {
+				[ name ]: blockType,
+			},
+		};
+
+		describe.each( [
```

Comment:
This thing (https://jestjs.io/docs/en/api#describeeachtable-name-fn-timeout) has a bit hard to follow API but once you learn it is awesome 👍 

I like how it keeps the readability of the `describe` and `it` interface but allows to remove some duplication.